src/character.h

```diff
@@ -764,6 +764,10 @@ class Character : public Creature, public visitable
 
         double recoil = MAX_RECOIL;
 
+        // Keep a count of moves passed in which resets every 100 turns as a result of practicing archery proficiency
+        // This is done this way in order to not destroy focus since `do_aim` is on a per-move basis.
+        int archery_aim_counter = 0;
```

Comment:
You're not saving this variable, so it will be reset after each save and load.
Also, we should probably look at making this proficiency not care about focus - it's about muscles, not knowledge. That doesn't need to happen here, though.